conanfile.txt

```diff
@@ -0,0 +1,6 @@
+[requires]
+openssl/1.1.1m
+asio/1.21.0
+
+[generators]
+json
```

Comment:
this will instruct conan to write the build information of the 2 libraries to a json which we can then parse in the setup.py